src/shared/codelens/localLambdaRunner.ts

```diff
@@ -538,10 +539,13 @@ export async function attachDebugger(params: {
         )
         isDebuggerAttached = await vscode.debug.startDebugging(undefined, params.debugConfig)
         numAttempts += 1
-        if (!isDebuggerAttached) {
+        if (isDebuggerAttached === undefined) {
+            isDebuggerAttached = false
+            shouldRetry = numAttempts < MAX_DEBUGGER_ATTEMPTS
+        } else if (!isDebuggerAttached) {
             retryDelay *= 2
 
-            shouldRetry = retryEnabled && retryDelay < 10000 ? true : false
+            shouldRetry = retryEnabled && (numAttempts < MAX_DEBUGGER_ATTEMPTS)
```

Comment:
Removing the retryDelay check here will still work but it'll make the `channelLogger.info` call later worthless.

Reply:
Disregard